flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/FlinkCatalog.java

```diff
@@ -90,6 +92,17 @@ public boolean databaseExists(String databaseName) throws CatalogException {
         return catalog.databaseExists(databaseName);
     }
 
+    @Override
+    public CatalogDatabase getDatabase(String databaseName) throws CatalogException {
+        try {
+            Database database = catalog.getDatabase(databaseName);
```

Comment:
I think we don't need to add `getDatabase` to `Catalog`, just `databaseExists` and return a empty `CatalogDatabase` for Flink Catalog is oK.